app/java/mvnw

```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
the link **is** `https`. When you type `http` it does take you to `https://`

I am not a lawyer but I think this part does not worth modification